Ensures users get updates for EME components on profile upgrade, due to non ideal settings having been used in the past
---
diff --git a/browser/components/ProfileDataUpgrader.sys.mjs b/browser/components/ProfileDataUpgrader.sys.mjs
index 0f397dc74e..e16e297b64 100644
--- a/browser/components/ProfileDataUpgrader.sys.mjs
+++ b/browser/components/ProfileDataUpgrader.sys.mjs
@@ -1097,6 +1097,21 @@ export let ProfileDataUpgrader = {
       if (!Services.prefs.prefHasUserValue(IPP_HAS_SEEN_FEATURE_PREF)) {
         Services.prefs.setBoolPref(IPP_HAS_SEEN_FEATURE_PREF, true);
       }
+
+      if (Services.prefs.prefHasUserValue("media.eme.enabled")) {
+        if (!Services.prefs.prefHasUserValue("media.gmp-manager.updateEnabled")) {
+          Services.prefs.setBoolPref("media.gmp-manager.updateEnabled", true);
+        }
+        if (!Services.prefs.prefHasUserValue("media.gmp-widevinecdm.enabled")) {
+          Services.prefs.setBoolPref("media.gmp-widevinecdm.enabled", true);
+        }
+      }
+
+      if (Services.prefs.prefHasUserValue("media.gmp-gmpopenh264.enabled")) {
+        if (!Services.prefs.prefHasUserValue("media.gmp-manager.updateEnabled")) {
+          Services.prefs.setBoolPref("media.gmp-manager.updateEnabled", true);
+        }
+      }
     }
 
     if (existingDataVersion < 181) {
